src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -62,13 +69,20 @@
 public class SkroutzGreekStemmer {
   public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
   private final CharArraySet stopwords;
+  private Map<String,char[]> step_0_exceptions;
 
   public SkroutzGreekStemmer(final CharArraySet stopwords) {
     this.stopwords = stopwords;
+    this.step_0_exceptions = new java.util.TreeMap();
```

Comment:
[minor] I would suggest importing the library and then using the class directly for two main reasons:

1. The import list on top of the code offers an overview of what we're using later on.
2. Java code is already a bit cluttered and loaded, it's a good idea to save some space from the actual logic.